### PR TITLE
Delete unused crate rlp_derive

### PR DESCRIPTION
**Pull request by leeyr338**
- delete unused crate rlp_derive

### CODE REVIEW

**Comment by boundless-forest** [comment]
rebase latest branch please.

**Comment by leeyr338** [comment]
> rebase latest branch please.

`git log` for my PR.
```shell
commit acf61760120fd2f5421c42af2416799f53983dcd (HEAD -> rm-gpl, origin/rm-gpl)
Author: leeyr <leeyr@cryptape.com>
Date:   Tue Aug 6 20:28:19 2019 +0800

    delete unused crate rlp_derive.

commit dfb41cf392f60ec147bebd7ab34777b2c23a9ba9
Author: kaikai1024 <kaikaifeng1024@gmail.com>
Date:   Mon Aug 5 15:17:34 2019 +0800

    Remove useless common of util.

commit 111c94aada524eea4bcdd0c406914d33e5fc1f97 (upstream/remove-gpl)
Merge: efddb53 284db7e
Author: Yaorong <38514341+leeyr338@users.noreply.github.com>
Date:   Tue Aug 6 13:55:36 2019 +0800

    Merge pull request #290 from kaikai1024/rm-nibble

    Remove nibble slice and vec.

commit 284db7eab48386300b39ad0d88b6e8dbf8f1963c
Author: kaikai1024 <kaikaifeng1024@gmail.com>
Date:   Tue Aug 6 10:50:38 2019 +0800

    Remove nibble slice and vec.
```

and the latest commit for branch `remove-gpl` is 111c94aada524eea4bcdd0c406914d33e5fc1f97. I thought I have rebase on the latest code.

And what's the problem from your view.

**Comment by boundless-forest** [comment]
> > rebase latest branch please.
> 
> `git log` for my PR.
> 
> ```shell
> commit acf61760120fd2f5421c42af2416799f53983dcd (HEAD -> rm-gpl, origin/rm-gpl)
> Author: leeyr <leeyr@cryptape.com>
> Date:   Tue Aug 6 20:28:19 2019 +0800
> 
>     delete unused crate rlp_derive.
> 
> commit dfb41cf392f60ec147bebd7ab34777b2c23a9ba9
> Author: kaikai1024 <kaikaifeng1024@gmail.com>
> Date:   Mon Aug 5 15:17:34 2019 +0800
> 
>     Remove useless common of util.
> 
> commit 111c94aada524eea4bcdd0c406914d33e5fc1f97 (upstream/remove-gpl)
> Merge: efddb53 284db7e
> Author: Yaorong <38514341+leeyr338@users.noreply.github.com>
> Date:   Tue Aug 6 13:55:36 2019 +0800
> 
>     Merge pull request #290 from kaikai1024/rm-nibble
> 
>     Remove nibble slice and vec.
> 
> commit 284db7eab48386300b39ad0d88b6e8dbf8f1963c
> Author: kaikai1024 <kaikaifeng1024@gmail.com>
> Date:   Tue Aug 6 10:50:38 2019 +0800
> 
>     Remove nibble slice and vec.
> ```
> 
> and the latest commit for branch `remove-gpl` is [111c94a](https://github.com/cryptape/cita-common/commit/111c94aada524eea4bcdd0c406914d33e5fc1f97). I thought I have rebase on the latest code.
> 
> And what's the problem from your view.

Maybe commit `Remove useless common of util` should not included in this pr.

**Comment by kaikai1024** [comment]
@leeyr338  I guess what @ouwenkg want to say is check the commits and should drop the dfb41cf392f60ec147bebd7ab34777b2c23a9ba9

File: util/src/lib.rs [hunk 1]
```diff
@@ -34,6 +34,7 @@ extern crate serde;
 extern crate toml;
 
 pub mod build_info;
+
```
Comment (kaikai1024) [inline util/src/lib.rs]: No need to change it.

Comment (leeyr338) [reply util/src/lib.rs]: okay.

**Review by boundless-forest — APPROVED** [review]

**Pull request merged**